### ISSUE TITLE
easyblock = line in easyconfigs should allow comments

### ISSUE THREAD

**Issue by JackPerdue** [hpcugent/easybuild-framework#2191]
If I comment the easyblock used for an easyconfig like:

```
easyblock = 'EB_QScintilla5'   # local (TAMU HPRC) easyblock
```

I get an error like:

```
ERROR: Failed to process easyconfig /general/software/noarch/easybuild/hprc/ec-dev/2017A-all/QScintilla5-2.10-GCCcore-6.3.0-Python-2.7.12-bare-PyQt-5.7.1.eb.terr: Failed to obtain class for EB_QScintilla5'   # local (TAMU HPRC) easyblock easyblock (not available?): No module named qscintilla5localtamuhprceasyblock
```

That ain't right.

I can write it as:

```
 # local (TAMU HPRC) easyblock
easyblock = 'EB_QScintilla5'
```

and it works but would prefer the former (and doesn't think adding a comment to that line should result in an error).

**Comment by boegel** [comment]
@JackPerdue that's indeed rather stupid, it's a matter of fixing `fetch_parameters_from_easyconfig` (https://github.com/hpcugent/easybuild-framework/blob/master/easybuild/framework/easyconfig/parser.py#L59) which now doesn't take into account comments at all...

**Comment by boegel** [comment]
fixed in #2192, which we can hopefully include in EasyBuild v3.2.0

**Comment by boegel** [comment]
#2192 was merged in time for EasyBuild v3.2.0, so closing this

Thanks for the bug report @JackPerdue!
